src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/FormContext.cs

```diff
@@ -29,6 +31,8 @@ public class FormContext
             }
         }
 
+        public bool HasFormData => _formData != null;
+
         public bool RenderedField(string fieldName)
```

Comment:
Properties before methods. The two `RenderedField()` overloads should remain at the bottom of this file.